docs/book/features/block_volume.md

```diff
@@ -147,90 +134,6 @@ This section describes the step by step instructions to provision a PersistentVo
-
-In a Tanzu Kubernetes Grid Service, the `StorageClass` gets automatically created and is made available to it by the
-underlying Supervisor cluster. An example storage class looks like -
-
-```yaml
-apiVersion: v1
-items:
-- apiVersion: storage.k8s.io/v1
-  kind: StorageClass
-  metadata:
-    name: gc-storage-profile
-  parameters:
-    storagePolicyID: 78b3d5e0-ab3f-4003-85ae-77c1b691a001
-  provisioner: csi.vsphere.vmware.com
-  reclaimPolicy: Delete
-  volumeBindingMode: Immediate
-kind: List
-metadata:
-  resourceVersion: ""
-  selfLink: ""
-```
-
-User only has to perform the following to dynamically provision a block volume:
-
-- Specify the desired `StorageClass` name in the `PersistentVolumeClaim` spec
-- Create a `PersistentVolumeClaim` in the namespace using the spec mentioned in the previous step
-- Wait for CNS to create a `PersistentVolume`
-- Verify if the status of `PersistentVolumeClaim` set to `Bound`
-
-### Dynamically provision a block volume in Supervisor Cluster<a id="dynamic_volume_provisioning_pacific"></a>
-
-In the case of `Supervisor` clusters, `StorageClass` creation happens automatically when a Storage Policy gets created.
-Following is an example of a `StorageClass` generated by one of the default storage policies within the cluster:
-
-```bash
-apiVersion: storage.k8s.io/v1
-kind: StorageClass
-metadata:
-  creationTimestamp: "2020-04-14T22:17:41Z"
-  name: wcpglobal-storage-profile
-  resourceVersion: "3477831"
-  selfLink: /apis/storage.k8s.io/v1/storageclasses/wcpglobal-storage-profile
-  uid: 0f84a03f-8a45-4116-8877-5ad0af28409c
-parameters:
-  storagePolicyID: 97aaf339-98c8-46d6-8c75-de49ea67f15d
-provisioner: csi.vsphere.vmware.com
-reclaimPolicy: Delete
-volumeBindingMode: Immediate
-```
-
-The user has to do the following to provision a `PersistentVolume`:
-
-- Create a new storage policy or add an existing storage policy of interest to the current namespace
-- Specify the `StoragePolicy` name (which got created by the storage policy) in the `PersistentVolumeClaim` spec
-- Create a `PersistentVolumeClaim` in the namespace using the spec mentioned in the previous step
-- Wait for CNS to create a `PersistentVolume`
-- Verify if the status of `PersistentVolumeClaim` set to `Bound`
-
-### Comparison of Cluster Flavor and the corresponding Storage Class Parameters<a id="dynamic_volume_provisioning_comparison"></a>
-
-   <table>
-    <thead>
-    <tr>
-    <th>Cluster Flavor</th>
-    <th>Storage Class Parameters</th>
-    </tr>
-    </thead>
-    <tbody>
-    <tr>
-    <td>Vanilla</td>
-    <td><em>storagepolicyname</em>, <em>datastoreURL</em>, <em>csi.storage.k8s.io/fstype</em></td>
-    </tr>
-    <tr>
-    <td>Supervisor</td>
-    <td><em>storagePolicyID</em></td>
-    </tr>
-    <tr>
-    <td>Tanzu Kubernetes Grid</td>
-    <td><em>storagePolicyID</em></td>
-    </tr>
-    </tbody>
-   </table>
-
 ## Static Volume Provisioning<a id="static_volume_provisioning"></a>
```

Comment:
Can you remove the in-line links provided in under this section on each heading as we are not using them anymore

Reply:
We use them, we have a listing at the top of the page - https://deploy-preview-769--kubernetes-sigs-vsphere-csi-driver.netlify.app/features/block_volume.html